### PR TITLE
First pass at the registry view

### PR DESCRIPTION
**Pull request by prajwalkshetty**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,9 +28,9 @@
         "preversion": "npm run build && npm run lint && npm run test && npm run check-extraneous && node scripts/preversion.js",
         "vscode:prepublish": "npm run build",
         "copy-schema": "cpx ./node_modules/brighterscript/bsconfig.schema.json ./dist -L",
-        "build": "npm run copy-schema && tsc -p ./",
+        "build": "npm run copy-schema && tsc -p ./ && npm run svelte-build",
         "lint": "tslint \"./src/**/*.ts\"",
-        "watch": "npm run copy-schema && tsc -w -p ./",
+        "watch": "npm run copy-schema && tsc -w -p ./ && npm run svelte-build",
```
Comment (triwav) [inline package.json]: Will this run? I think the tsc will prevent the build until it's done watching

Comment (prajwalkshetty) [reply package.json]: Sorry that was some testing thing will remove it

Comment (prajwalkshetty) [reply package.json]: it did not throw any error though

File: src/RDBViewProviders.ts [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
 import { OnDeviceComponent } from 'roku-test-automation';
 
 export class RDBRegistryViewProvider implements vscode.WebviewViewProvider {
     private view?: vscode.WebviewView;
     private odc?: OnDeviceComponent;
+    private extensionPath: string;
 
-    constructor(context: vscode.ExtensionContext, private rdbOutputChannel: vscode.OutputChannel) {}
+    constructor(context: vscode.ExtensionContext, private rdbOutputChannel: vscode.OutputChannel) {
+        this.extensionPath = context.extensionPath + "/dist/ui/rdb"
```
Comment (triwav) [inline src/RDBViewProviders.ts]: Feel like it makes sense to name this something different for clarification. Maybe this.rdbBasePath

File: src/RDBViewProviders.ts [hunk 3]
```diff
@@ -1,19 +1,25 @@
 import * as vscode from 'vscode';
+import * as path from 'path';
```
Comment (triwav) [inline src/RDBViewProviders.ts:2]: Since the rest of the project uses spaces, can we standardize on spaces for spacing

**Review by triwav — APPROVED** [review]

**Review by triwav — APPROVED** [review]

**Pull request merged**